### ISSUE TITLE
Speed of switching orientation from 'Movement direction' to 'Compass directon'

### ISSUE THREAD

**Issue by IgorEliezer** [osmandapp/Osmand#7130]
This issue is both a question and a translation report.

**1. Question:**

In Navigation Settings we have "Select speed of switching orientation from 'Movement direction' to 'Compass direction' below.", how does it work?

My guess is that, if you moving slower than the selected speed the map orientation switches from 'Movement direction' to 'Compass direction'. Does this also hold true in the opposite? If faster than the selected speed the map orientation switches from 'Compass direction' to 'Movement direction'.

**2. Source translation report:**

Ref: `map_orientation_change_in_accordance_with_speed_descr`.

a. In Android version, direction is misspelled: "Select speed of switching orientation from 'Movement direction' to 'Compass **_directon_**' below."

b. The iOS version diverges from Android version:

"Select below which speed the map orientation switches from 'To direction of movement' to 'To compass'"

**Comment by vshcherb** [comment]
Yes it switches back in both cases. I think the functionality is exactly the same

**Comment by vshcherb** [comment]
![image](https://user-images.githubusercontent.com/1042025/60734149-4297fb00-9f4f-11e9-86c1-bbb0829b6ae6.png)
